### ISSUE TITLE
Add section for requests

### ISSUE THREAD

**Issue by timelyportfolio** [hafen/htmlwidgetsgallery#8]
I am not sure it fits with the gallery, but it would be nice if there was a Requests section on the galley or somewhere else.

**Comment by hafen** [comment]
Indeed.  @ramnathv mentioned the idea of having a google spreadsheet of widgets people are working on, which could also handle requests.

**Comment by DataStrategist** [comment]
+1. Any progress on this?
